### PR TITLE
Misc/update contribution workflow

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,29 +2,31 @@
...
-The report of issues and the proposal of new features is very welcome.
-Please use the GitHub issue page for this.
+Issue reports and feature proposals are very welcome.
+Please use the [GitHub issue page](https://github.com/glotzerlab/signac-flow/issues/) for this.
 
 ## Contributing code
 
-Code contributions to the signac-flow open-source project are welcomed via pull requests on GitHub.
-Prior any work you should contact the signac-flow developers to ensure that the planned development meshes well with the directions and standards of the project.
+Code contributions to the signac open-source project are welcomed via pull requests on GitHub.
```
Comment (bdice) [inline CONTRIBUTING.md]: Do we want this to say `signac-flow` or not? (is it the package or the framework?)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,29 +2,31 @@
...
+Prior any work you should contact the signac developers to ensure that the planned development meshes well with the directions and standards of the project.
 All contributors must agree to the Contributor Agreement ([ContributorAgreement.md](ContributorAgreement.md)) before their pull request can be merged.
 
-General guidelines:
+### General guidelines
 
-  * Use the [git flow][git flow] model of development:
-    - New features should be developed in a feature branch based on `develop`.
+  * Use the [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) model of development:
```
Comment (bdice) [inline CONTRIBUTING.md]: This description of "git flow" is accurate for the model of development we've been using, except for possibly release branches, right? (re: potentially changing to "OneFlow" conversation in Slack)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,29 +2,31 @@
...
-  * Write API documentation in this package, and put usage information, guides, and concept overviews in the [framework documentation](https://docs.signac.io/) ([source][signac-docs-src]).
+  * Write API documentation in this package, and put usage information, guides, and concept overviews in the [framework documentation](https://docs.signac.io/) ([source](https://github.com/glotzerlab/signac-docs/)).
+
+Please see the [Support](https://docs.signac.io/projects/signac-flow/en/latest/support.html) section as part of the documentation for detailed development guidelines.
+
+### Code Style
 
-[signac-docs-src]: https://github.com/glotzerlab/signac-flow
-[gitflow]: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
+Code submitted to the signac project must adhere to the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/) with the exception that lines may have up to 100 characters.
```
Comment (bdice) [inline CONTRIBUTING.md]: Same question as above about signac-flow package vs signac framework.

File: CONTRIBUTING.md (hunk 1 above)
Comment (csadorf) [reply CONTRIBUTING.md]: I intentionally just left it general there..

File: CONTRIBUTING.md (hunk 2 above)
Comment (csadorf) [reply CONTRIBUTING.md]: I did not touch this. We should clarify our branching model and then updates the guidelines on all affected repositories.

File: CONTRIBUTING.md (hunk 3 above)
Comment (csadorf) [reply CONTRIBUTING.md]: This one makes me wonder... I'll just switch it to signac-flow in both cases for consistency.

**Pull request merged**